cmd/fuzzbinder/README.md

```diff
@@ -1 +1,66 @@
 # Fuzzing Claims for revisions of a source code
+
+The goal of Fuzzbinder is to generate, sign and publish (in a transparency log) fuzzing claims for revisions of the source code of a given project.
+
+By following the steps described below, you will be able to generate fuzzing claims for your open-source project using the fuzzing reports generated by OSS-Fuzz and ClusterFuzz for your project, and the command line tool
+[`cmd/fuzzbinder`](/cmd/fuzzbinder/) that we have provided to generate fuzzing claims.
+
+## Generating fuzzing claims using the `cmd/fuzzbinder` tool
+
+If you have not already done so, consult the [`development guidelines`](./../../docs/development-guidelines.md) to find out what you need to install before following the steps below.
+
+### Step 1: Establish access to Google Cloud Storage
+
+To get the fuzzing statistics, you should have access to the fuzzing metadata generated by OSS-Fuzz and ClusterFuzz for your project. Set up `gsutil` and ensure that you have access to the fuzzing metadata by doing the following:
+
+- Install the [gsutil tool](https://cloud.google.com/storage/docs/gsutil_install).
+- Check whether you have access to the coverage reports and the fuzzing logs  of your project:
+
+```sh
+$ gsutil ls gs://oss-fuzz-coverage/${PROJECT_NAME}
+```
+
+```sh
+$ gsutil ls gs://${PROJECT_NAME}-logs.clusterfuzz-external.appspot.com
+```
+
+Note that the coverage reports in `gs://oss-fuzz-coverage` are public. However, you need to be a primary_contact or an auto_cc for the project for which you want to generate fuzzing claims to have access to the fuzzing logs in `gs://${PROJECT_NAME}-logs.clusterfuzz-external.appspot.com` of that project. If you are not sure whether you are a primary_contact or an auto_cc for the project, see this [OSS-Fuzz documentation](https://google.github.io/oss-fuzz/getting-started/new-project-guide/#primary) for more details.
+
+If you are a primary_contact or an auto_cc and you get an authorization error from the commands above, run the following commands:
+
+```sh
+$ gcloud auth application-default login
+```
+
```

Comment:
It he next step required too?

Also, to complete the command above you have to follow a link to a browser and copy the token. Is not there a better way to authenticate?

Reply:
> Is the next step required too?

I agree, I did tests and the next step is not required. 

> Also, to complete the command above you have to follow a link to a browser and copy the token. Is not there a better way to authenticate?

I just had to connect via email in my tests. This is the command to run in [this documentation](https://googleapis.dev/python/google-api-core/latest/auth.html) for authentication. I linked this in this readme also for more details. However, I don't think that they mention something simpler.